tests/test_align.py

```diff
@@ -1,240 +0,0 @@
-from copy import deepcopy
-
-import numpy as np
-import pytest
-from rdkit import Chem
-from rdkit.Chem import AllChem
-
-from timemachine.constants import DEFAULT_FF
-from timemachine.fe.atom_mapping import get_core_by_mcs, get_core_with_alignment, mcs
-from timemachine.fe.topology import AtomMappingError
-from timemachine.fe.utils import set_romol_conf
-from timemachine.ff import Forcefield
-from timemachine.md import align
```

Comment:
Is deleting this test file intentional?

(Wasn't deleted in initial version I initially reviewed, and some code it tests e.g. `md/align.py` was not removed by this PR)

Reply:
I'm not sure how to make this compatible with the alignment code yet - I'll talk to @badisa and @jkausrelay later this week to decide how to deal with this correctly (it had a bunch of tests that I wasn't sure how to fix)